app/models/resource.py

```diff
@@ -78,7 +79,7 @@ def __repr__(self):
         return '<Descriptor \'%s\'>' % self.name
 
 
-class Resource(db.Model):
+class ResourceBase(db.Model):
     """
     Schema for resources with relationships to descriptors.
```

Comment:
change comment to reflect "base class that contains items that approved and suggested resources have "